test/cpp/mixer_test.cpp

```
@@ -13,82 +13,148 @@
+ *
+ * Standalone CI mirror of android/app/src/main/cpp/audio_mixer.{cpp,h}.
+ * Keep the [onVoiceFrame] / poison logic in lock-step with production —
+ * this file is what CI exercises (production needs the Android NDK).
  */
 class AudioMixer {
+public:
+    // Mirrors production AudioMixer::kPoisonThreshold (audio_mixer.h).
+    static constexpr uint32_t kPoisonThreshold = 16;
+
 private:
+    struct DeviceState {
```

Clang (14.0.6)

[warning] constructor does not initialize these fields: buffer

(cppcoreguidelines-pro-type-member-init)